README.md

```diff
@@ -2,9 +2,9 @@
 
 ## About
 
-This project is an App Calculator using React
+This project is an App Calculator using React.
 
-<!-- ![screenshot](./src/assets/Todo-list.png) -->
+![screenshot](./src/assets/static/Screenshot.png)
```

Comment:
- Update the `To get a local copy of this repo` instruction. Step 1 has this instruction: `$ cd "here comes the path of your selected directory (double quotes must not be here)"`. This instruction is not clear and doesn't look useful to a user who wants to checkout the app locally. Always be very explicit in your instructions.